### PR TITLE
updates for /vc addition

### PR DESCRIPTION
**Pull request by kl5pierce**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A: built HTML on the command line of Linux Mint VM, and previewed using Firefox web browser.
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

Note: Unchecked marks were considered N/A, due to updates being for a .rst file for Read The Docs, and not code for VOLTTRON itself.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/VOLTTRON/volttron/pull/1807%23discussion_r219638955%22%2C%20%22https%3A//github.com/VOLTTRON/volttron/pull/1807%23discussion_r219644389%22%2C%20%22https%3A//github.com/VOLTTRON/volttron/pull/1807%23discussion_r219644710%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20f002e267eee080883d2ad30a30b03816cffb46b2%20docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst%2014%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/VOLTTRON/volttron/pull/1807%23discussion_r219638955%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22this%20one%20should%20stay%20as%20/discovery/%20as%20it%27s%20not%20a%20vc%20specific%20command...this%20one%20is%20available%20to%20any%20caller.%22%2C%20%22created_at%22%3A%20%222018-09-21T22%3A06%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/3979063%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/craig8%22%7D%7D%2C%20%7B%22body%22%3A%20%22done%22%2C%20%22created_at%22%3A%20%222018-09-21T22%3A42%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/14077072%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kl5pierce%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst%3AL46-53%22%7D%2C%20%22Pull%20f002e267eee080883d2ad30a30b03816cffb46b2%20docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/VOLTTRON/volttron/pull/1807%23discussion_r219644710%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Done%22%2C%20%22created_at%22%3A%20%222018-09-21T22%3A45%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/14077072%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/kl5pierce%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst%3AL150-157%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull f002e267eee080883d2ad30a30b03816cffb46b2 docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst 14'></a> &nbsp;<b><a href='https://github.com/VOLTTRON/volttron/pull/1807#discussion_r219638955'>File: docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst:L46-53</a></b>
- <a href='https://github.com/craig8'></a> this one should stay as /discovery/ as it's not a vc specific command...this one is available to any caller.
- <a href='https://github.com/kl5pierce'></a> done
- [ ] <a href='#crh-comment-Pull f002e267eee080883d2ad30a30b03816cffb46b2 docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst 5'></a> &nbsp;<b><a href='https://github.com/VOLTTRON/volttron/pull/1807#discussion_r219644710'>File: docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst:L150-157</a></b>
- <a href='https://github.com/kl5pierce'></a> Done

<a href='https://www.codereviewhub.com/VOLTTRON/volttron/pull/1807?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/VOLTTRON/volttron/pull/1807?approve=1'></a>&nbsp;<a href='https://github.com/VOLTTRON/volttron/pull/1807'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by craig8** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/VOLTTRON/volttron/pull/1807'>Refresh</a>.<a href='#crh-update-none'></a>

File: docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ Verify that the instance is serving properly by pointing your web browser to
 
 ::
 
-    http://127.0.0.1:8080/discovery/
+    http://127.0.0.1:8080/vc/discovery/
```
Comment (craig8) [inline docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst]: this one should stay as /discovery/ as it's not a vc specific command...this one is available to any caller.

File: docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst [hunk 2]
```diff
@@ -150,7 +150,7 @@ details on how to configure the agent for your specific use case.
 Log In
 ------
 
-To log in to VOLTTRON Central, navigate in a browser to localhost:8080,
+To log in to VOLTTRON Central, navigate in a browser to localhost:8080/vc,
```
Comment (craig8) [inline docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst]: should be localhost:8080/vc/index.html

redirection doesn't work now :(

File: docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst (hunk 1 above)
Comment (kl5pierce) [reply docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst]: done

File: docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst (hunk 2 above)
Comment (kl5pierce) [reply docs/source/devguides/walkthroughs/VOLTTRON-Central-Demo.rst]: Done

**Pull request merged**